lib/Zonemaster/Backend/RPCAPI.pm

```diff
@@ -57,29 +60,52 @@ sub new {
             die "$@ \n" if $@;
             $self->{db} = $backend_module->new( { config => $config } );
         };
-        die "$@ \n" if $@;
+        if ($@) {
+            warn "Internal error #002: Failed to initialize the database backend module: [$@] \n";
+            die "Internal error #002 \n" if $@;
+        }
```

Comment:
Why not a call to handle_exception() here so that all cases are done in the same way?

Reply:
Fixed in dcf35a6